[2022-01-11 20:17:28,437] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_03 scheduled__2022-01-05T00:00:00+00:00 [queued]>
[2022-01-11 20:17:28,480] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_03 scheduled__2022-01-05T00:00:00+00:00 [queued]>
[2022-01-11 20:17:28,483] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:17:28,483] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-11 20:17:28,483] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:17:28,556] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_03> on 2022-01-05 00:00:00+00:00
[2022-01-11 20:17:28,565] {standard_task_runner.py:52} INFO - Started process 3597 to run task
[2022-01-11 20:17:28,613] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_03', 'scheduled__2022-01-05T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpcckfusvg', '--error-file', '/tmp/tmpjea0emwl']
[2022-01-11 20:17:28,643] {standard_task_runner.py:77} INFO - Job 37: Subtask task_03
[2022-01-11 20:17:34,367] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_03 scheduled__2022-01-05T00:00:00+00:00 [running]> on host airflowvm
[2022-01-11 20:17:35,008] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_03
AIRFLOW_CTX_EXECUTION_DATE=2022-01-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-05T00:00:00+00:00
[2022-01-11 20:17:35,027] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-11 20:17:35,034] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-01-11 20:17:35,061] {subprocess.py:85} INFO - Output:
[2022-01-11 20:17:38,070] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-11 20:17:38,255] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_03, execution_date=20220105T000000, start_date=20220111T201728, end_date=20220111T201738
[2022-01-11 20:17:38,356] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 20:17:38,653] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
